Specification/2._Basic_concepts.md

```diff
@@ -67,17 +67,17 @@ allocation procedure.
 
 ## Service invocation
 
-Service invocation is a two-step data exchange between exactly two nodes: the client and the server.
+Service invocation is a two-step data exchange between exactly two nodes: a client and a server.
 Service invocation is performed in three steps:
 
 1. The client sends a *service request* to the server.
 2. The server application takes appropriate actions and returns the response data.
 3. The server sends a *service response* to the client.
 
-Typical use cases for this type of communication include the following: node configuration parameter update,
+Typical use cases for this type of communication include: node configuration parameter update,
 firmware update, momentary action request, file transfer, and other service tasks.
```

Comment:
An aperiodic or ad hoc action request. If it seems too confusing, this item can be removed completely.